app/queries/placements/placements_query.rb

```diff
@@ -8,7 +8,7 @@ def call
     scope = partner_school_condition(scope)
     scope = subject_condition(scope)
     scope = establishment_group_condition(scope)
-    scope = only_available_placements_condition(scope)
+    scope = placements_to_show(scope)
```

Comment:
Should this method name be suffixed with `_condition` for consistency?

```suggestion
    scope = placements_to_show_condition(scope)
```